### ISSUE TITLE
Allow sliding sync

### ISSUE THREAD

**Issue by getzze** [YunoHost-Apps/conduit_ynh#19]
I tried connecting from the new Element X (beta) Android app but it complains that you need to have a sliding-sync server.
Sliding sync is included in Conduit, you only need to provide the address of the server, which is the same as Conduit:
https://gitlab.com/famedly/conduit/-/issues/398

I think it would work by adding a line in [server_name.conf](https://github.com/YunoHost-Apps/conduit_ynh/tree/maste/conf/server_name.conf), like this:
```
location /.well-known/matrix/client {
    return 200 '{
        "m.homeserver": { "base_url": "https://__DOMAIN__" },
        "org.matrix.msc3575.proxy": {"url":"https://__DOMAIN__" },
        "im.vector.riot.jitsi": {"preferredDomain": "jitsi.riot.im"},
        "im.vector.riot.e2ee": {"default": true }
    }';
...
}
```

The `server_name.conf` was not installed at installation actually, I don't know why...
I made it work by adding an alias to `location /.well-known/matrix/client` in `/etc/nginx/conf.d/mydomain.conf` after installation.

**Comment by getzze** [comment]
I created the `server_name.conf` with the same content as in the source files with the added line with `"org.matrix.msc3575.proxy"` and ElementX recognizes that sliding-sync is installed. So this is working.

However I am stuck with the `Setting up your account.` message, as described in this issue: https://gitlab.com/famedly/conduit/-/issues/382

**Comment by getzze** [comment]
Update on the Sliding sync support in Conduit, it should work in the next release:
https://gitlab.com/famedly/conduit/-/issues/424

**Comment by ericgaspar** [comment]
Is the issue fixed with 0.7.0 release?

**Comment by getzze** [comment]
Yes, I can connect to my conduit server from ElementX.
